src/config.rs

```diff
@@ -9,8 +9,12 @@ pub struct Config {
     #[arg(long, value_name = "URL", default_value = "http://localhost:5052")]
     pub beacon_node: String,
 
+    /// Address to listen on.
+    #[arg(long, value_name = "ADDRESS", default_value = "0.0.0.0")]
```

Comment:
I think we should keep this as localhost, otherwise it's a bit of a security liability.

```suggestion
    #[arg(long, value_name = "ADDRESS", default_value = "127.0.0.1")]
```

Reply:
Nice, thanks for fixing the docs as well! 🙏